### ISSUE TITLE
Bug with hardsuits (Perhaps all equippables?)

### ISSUE THREAD

**Issue by chaoko99** [tgstation/tgstation#21457]
[Note]: # (State what the issue is from a "whats wrong" prospective. Issue reports should clearly allow maintainers to understand whats wrong and how to test/reproduce if that is not obvious. Avoid ambiguity. Start your issue report below both of these lines (or remove them))

[Admins]: # (If you are reporting a bug that occured AFTER you used varedit/admin buttons to alter an object out of normal operating conditions, please verify that you can re-create the bug without the varedit usage/admin buttons before reporting the issue.)

http://puu.sh/sd3vD/df20c26a89.png

I've found that one can, in fact, drag the toggle icons onto their inventory, and even activate them.

**Comment by PKPenguin321** [comment]
Doesn't look like a bug to me, you're just dragging one hud element over another.

**Comment by ShadowDeath6** [comment]
this happens with all action buttons, and if it's a nodrop item there can be some problems since it's impossible to remove the action button without removing the item itself

**Comment by jughu** [comment]
If you are stupid enough to put your own action button over your hand rendering it unusable it is your own fault honestly

**Comment by WJohn** [comment]
Why can we even drag action buttons? It doesn't even snap to a tile grid so it's almost always gonna be off center and you can also end up with cases like this.

**Comment by SgtxRootbeer** [comment]
we should be able to maybe lock the action buttons (to a grid maybe?) and bind the function keys to them (unless the f-keys are already in use, in that case, maybe alt+f# or alt+#?)
hotkeys are fun but action buttons seem like a missed oppritunity.

**Comment by phil235** [comment]
@RemieRichards

**Comment by RemieRichards** [comment]
I don't see the issue? there's the reset toggle for this, if you shift-click an action button it goes back to the top left, realigning as if you never moved it.

grid snapping support is already there, but it's only enabled on an (unused) screen object type, I don't -really- see any issues with being able to toggle grid snapping, and probably toggle their movement altogether (probably with a cute little padlock icon)

tl;dr not bugged, but there's some good suggestions for improvements here.

**Comment by Dorsisdwarf** [comment]
I think that we made a huge mistake when we made action buttons not be /snap

**Comment by lzimann** [comment]
You can shift click the action button to reset its location by the way.
